### ISSUE TITLE
Small text on homepage doesn't meet accessibility

### ISSUE THREAD

**Issue by mark-a-wilson** [bcgov/digital_marketplace#167]
From [GDX accessibility audit](https://bcgov.sharepoint.com/:b:/t/00109-digital.gov.bc.ca/EQrwcYPvVqNPrpsUS_cmXPMBtgckjCybC3Nm02RqwMbRLQ?e=Jqz9Da) conducted in July 2021.

#### Issue
This text beneath the numbers is very small and difficult to read, particularly for those with low vision:
![Screen Shot 2021-08-16 at 1 40 31 PM](https://user-images.githubusercontent.com/16946297/129630902-132e9d89-d80a-4c85-b261-0c9d594c3505.png)

#### Suggestion
Increase text size to at least 14px
